docs/content/release_notes/supported_orchestration.md

```diff
@@ -4,11 +4,13 @@ The following table details orchestration platforms suitable for deployment of t
 
 |Orchestration platform|Version|Architecture|
 |----------------------|-------|------------|
-|Kubernetes|1.21|x86|
 |Kubernetes|1.22|x86|
+|Kubernetes|1.23|x86|
```

Comment:
@roysahar-ibm is that correct?

Reply:
@rivkap we are supporting 1.21, 1.22

Reply:
@roysahar  Please confirm that this has changed since [PR 399](https://github.com/IBM/ibm-block-csi-driver/pull/399) (CSI-3508)

Reply:
confirmed during meeting. done